src/cosmosdb-preview/azext_cosmosdb_preview/vendored_sdks/azure_mgmt_cosmosdb/version.py

```diff
@@ -9,5 +9,5 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "2.0.0rc3"
+VERSION = "3.0.0"
```

Comment:
You can name the sdk version as 3.0.0.rc1 (if this version would have been released it will be called this)

Reply:
rc: release candidate (since this is a  preview apiversions)